prog/vm/github_runner.rb

```diff
@@ -24,15 +24,14 @@ def self.assemble(installation, repository_name:, label:, default_branch: nil)
   end
 
   def pick_vm
-    skip_sync = true
-    prefer_performance = github_runner.installation.project.get_ff_performance_runners
+    prefer_performance = (github_runner.installation.allocator_preferences["family_filter"] || []).include?("performance")
```

Comment:
```suggestion
    prefer_performance = github_runner.installation.allocator_preferences["family_filter"]&.include?("performance")
```

Since this is only used in boolean context, the `nil`/`false` distinction doesn't matter here.

Reply:
I changed it and added a new test for branch coverage as well.